src/helpers/getAPIReadable.js

```diff
@@ -0,0 +1,3 @@
+// Returns the API readable value based on the human-readable text
```

Comment:
I think this can be either removed or renamed. I think helpers should describe generally what they do and not specifically you intend to use it. This would be something like "findKeyForValue'

Reply:
Removed, thank you!